app/src/main/java/nz/org/cacophony/birdmonitor/views/BirdCountActivity.java

```diff
@@ -153,15 +153,13 @@ public void recordNow() {
             long durationInSeconds = 0;
 
             if (rbFiveMinute.isChecked()) {
-                myIntent.putExtra("type", "birdCountButton5");
-                durationInSeconds = Util.getRecordingDuration(getApplicationContext(), "birdCountButton5");
+                myIntent.putExtra("type", Prefs.BIRD_COUNT_5_ALARM);
             } else if (rbTenMinute.isChecked()) {
-                myIntent.putExtra("type", "birdCountButton10");
-                durationInSeconds = Util.getRecordingDuration(getApplicationContext(), "birdCountButton10");
+                myIntent.putExtra("type",  Prefs.BIRD_COUNT_10_ALARM);
             } else if (rbFifteenMinute.isChecked()) {
-                myIntent.putExtra("type", "birdCountButton15");
-                durationInSeconds = Util.getRecordingDuration(getApplicationContext(), "birdCountButton15");
+                myIntent.putExtra("type",  Prefs.BIRD_COUNT_15_ALARM);
             }
+            durationInSeconds = Util.getRecordingDuration(getApplicationContext(),  Prefs.BIRD_COUNT_5_ALARM);
```

Comment:
Won't this always get a duration of 5 seconds?

Reply:
Yep thanks, for checking. Fixed now